com.github.giacomogroppi.writernote-qt.json

```
@@ -0,0 +1,78 @@
+{
+  "app-id": "com.github.giacomogroppi.writernote-qt",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.2",
+  "sdk": "org.kde.Sdk",
+  "command": "writernote",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--share=ipc"
+  ],
+  "cleanup": [
+    "/lib/pkgconfig",
+    "/share/man",
+    "*.la"
```

```suggestion
   "/include",
    "*.la",
    "*.a"
```

I'll assume that if there are `.la` there are `.a`

you should also cleanup the includes